src/test/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcerTest.java

```diff
@@ -55,19 +47,24 @@ void test_enforce_version(AncestorBlockUpdater ancestorBlockUpdater, ReleaseRequ
         verify(ancestorBlockUpdater, times(1)).ensureAncestorBlockInPosition(any());
     }
 
+    @Test
+    void enforce_version_three_fail() {
+        Assertions.assertThrows(ReleaseRequirementsEnforcerException.class, () -> {
+            releaseRequirementsEnforcer.enforce(3, mock(ReleaseCreationInformation.class));
+        }, "Unsupported version 3");
+    }
+
     @Test
     void enforce_version_two_updater_fails() throws Exception {
-        AncestorBlockUpdater ancestorBlockUpdater = mock(AncestorBlockUpdater.class);
-        doThrow(new Exception()).when(ancestorBlockUpdater).ensureAncestorBlockInPosition(any());
-        ReleaseRequirementsEnforcer enforcer = new ReleaseRequirementsEnforcer(ancestorBlockUpdater);
+        AncestorBlockUpdater blockUpdater = mock(AncestorBlockUpdater.class);
+        doThrow(new Exception()).when(blockUpdater).ensureAncestorBlockInPosition(any());
+        ReleaseRequirementsEnforcer enforcer = new ReleaseRequirementsEnforcer(blockUpdater);
```

Comment:
```suggestion
      doThrow(new Exception()).when(ancestorBlockUpdater).ensureAncestorBlockInPosition(any());
```

no need to re-declare variables

Reply:
Solved